bsp/esp-box-lite/idf_component.yml

```diff
@@ -9,7 +9,7 @@ dependencies:
   idf: ">=4.4.5"
 
   button:
-    version: "^2"
+    version: "^3"
```

Comment:
Why do we need to bump this version requirement?

Reply:
IDF-component-manager will automatically select latest compatible version. We shouls still support 2.* versions. We can do something like `>=2`

Reply:
> IDF-component-manager will automatically select latest compatible version. We shouls still support 2.* versions. We can do something like `>=2`

OK, thanks, I'll check and update it.

Reply:
> Why do we need to bump this version requirement?

Because I'll pull three BSPs together (esp-box, esp-box-lite, esp-box-3).
For esp-box-3, I need the feature in lastest version(>=3), there was a conflict here and I used a local components before. I want to keep the same here, ans  I can delete the local btn components.